dashboard/app/views/home/_homepage.html.haml

```diff
@@ -42,3 +42,20 @@
       = link_to pd_workshop_admins_path do
         %h3
           Directory for Workshop Admins
+    - courses_facilitated = Pd::CourseFacilitator.where(facilitator: current_user).map(&:course)
+    -if courses_facilitated.any?
+      %h1
+        Facilitator Landing Page
+
+      -if courses_facilitated.include? Pd::Workshop::COURSE_CSF
+        %a{href: "https://curriculum.code.org/plcsf/", target: '_blank'}
+          %h3
+            CSF Facilitator Landing Page
+      -if courses_facilitated.include? Pd::Workshop::COURSE_CSD
+        %a{href: "https://curriculum.code.org/plcsd/", target: '_blank'}
```

Comment:
I feel like we should be using a helper function (or Rails route) to refer to `"https://curriculum.code.org"` rather than literally mentioning it multiple times.  Amongst other advantages, it would make it much easier to change this domain in the future in one place, and also to link to a variation of it for different environments such as `"staging"` and `"test"`...

Reply:
I see your point but I'm going to leave it here since there are already lots of specific references to curriculum.code.org. Instead I will take a work item as part of platformization stuff to do this clean up.